xstate-chatbot/nodejs/src/machine/vitals.js

```diff
@@ -615,15 +758,18 @@ const vitalsFlow = {
               }),
               always: [
                 {
-                  cond: (context) => context.grammer == dialog.INTENTION_UNKOWN,
-                  target: 'error'
-                },
-                {
+                  cond: (context, event) => context.slots.vitals.com_status == '1',
                   actions: assign((context, event) => {
-                    context.slots.vitals.symptoms.respiratoryIssues = context.intention;
+                    console.log(context.slots.vitals.com_status)
                   }),
-                  target: '#comorbidities'
-                }
+                  target: '#heartrelated'
+                },
+              {
+                cond: (context, event) => context.slots.vitals.com_status == '0',
+                actions: assign((context, event) => {
+                  context.slots.vitals.symptoms.respiratoryIssues = context.intention;                }),
```

Comment:
Same statement is needed in the previous condition where you have written console log.

Reply:
updated